apps/_pomodoro_app/_realtime_tests/store.test.js

```diff
@@ -0,0 +1,27 @@
+const Store = require('../realtime/application/Store')
```

Comment:
it looks like `Store.spec.js` 
Is it a part of realtime_tests?

Reply:
It looks like I don't know about `spec.js` and haven't even thought of it

Reply:
Done something with it, but not sure if everything is correct.